cherrypy/_cpwsgi.py

```diff
@@ -54,33 +53,36 @@ class VirtualHost(object):
 
         cherrypy.tree.graft(vhost)
     """
-    default = None
-    """Required.
-
-    The default WSGI application.
-    """
 
+    default = None
     use_x_forwarded_host = True
-    """If True (the default), any "X-Forwarded-Host"
-    request header will be used instead of the "Host" header. This
-    is commonly added by HTTP servers (such as Apache) when proxying."""
-
     domains = {}
```

Comment:
```suggestion
    domains = {}
    """A dict of {host header value: application} pairs.
    The incoming "Host" request header is looked up in this dict, and,
    if a match is found, the corresponding WSGI application will be
    called instead of the default. Note that you often need separate
    entries for "example.com" and "www.example.com". In addition, "Host"
    headers may contain the port number.
    """
```